packages/scripts/scripts/build-blocks-manifest.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { sync: glob } = require( 'fast-glob' );
+const json2php = require( 'json2php' );
+const chalk = require( 'chalk' );
+
+/**
+ * Internal dependencies
+ */
+const { getArgFromCLI } = require( '../utils' );
+
+// Set default paths
+const defaultInputDir = 'build';
+const defaultOutputFile = path.join( 'build', 'blocks-manifest.php' );
+
+// Parse command line arguments
+const inputDir = getArgFromCLI( '--input' ) || defaultInputDir;
+const outputFile = getArgFromCLI( '--output' ) || defaultOutputFile;
+
+// Find all block.json files
+const blockJsonFiles = glob( './**/block.json', {
+	cwd: path.resolve( process.cwd(), inputDir ),
+	absolute: true,
+} );
```

Comment:
The script should first check if the `build` directory exists and abort with error if it doesn't.

It makes sense to run this script only if the blocks were already built and the `build` directory was created. If you forgot to run the build first, you should be told about it, instead of silently creating an empty manifest file.

Reply:
:+1: Check https://github.com/WordPress/gutenberg/pull/65866/commits/e673f314c2f0cd3db49471c2892257889d4bff56